sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -142,8 +145,12 @@ class SessionCatalog(
   // ----------------------------------------------------------------------------
 
   def createDatabase(dbDefinition: CatalogDatabase, ignoreIfExists: Boolean): Unit = {
-    val qualifiedPath = makeQualifiedPath(dbDefinition.locationUri).toString
     val dbName = formatDatabaseName(dbDefinition.name)
+    if (dbName == GLOBAL_TEMP_VIEW_DATABASE) {
```

Comment:
Is this too restrictive? If the user ask directly for  `GLOBAL_TEMP_VIEW_DATABASE `, may be we should allow it?